### PR TITLE
Add version to addresses files

### PR DESCRIPTION
**Pull request by iainnash**

### CODE REVIEW

**Review by kulkarohan — APPROVED** [review]

**Comment by oveddan** [comment]
would be better as:
CONTRACT_1155_IMPL: {
   address: 0x....,
   version: "1.4.0"
}

this would make it easier to consume in the sdk

**Review by oveddan — CHANGES_REQUESTED** [review]
please see comment about formatting

**Comment by oveddan** [comment]
can we also have the deployment scripts populating the values, instead of having to manually set them? manually setting seems precarious

**Comment by oveddan** [comment]
can you verify that the script `node script/copy-deployed-contracts.mjs deploy` still works as expected?

File: addresses/420.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "CONTRACT_1155_IMPL": "0x34c976E09EEcA565E208b0860Ea83F3cB881d205",
+  "CONTRACT_1155_IMPL_VERSION": "1.4.0",
```
Comment (kolber) [inline addresses/420.json:3]: 🙌

**Review by oveddan — APPROVED** [review]
please fix build then SGTM.  lets look at a better way to tag versions in the future

**Pull request merged**